st2reactor/st2reactor/rules/filter.py

```diff
@@ -54,7 +70,9 @@ def filter(self):
 
         payload_lookup = PayloadLookup(self.trigger_instance.payload)
 
-        LOG.debug('Trigger payload: %s', self.trigger_instance.payload)
+        LOG.debug('Trigger payload: %s', self.trigger_instance.payload,
```

Comment:
True, I didn't really touch the log level though, I just added additional info to the context.